spark-indexer/src/main/java/org/uniprot/store/spark/indexer/taxonomy/reader/TaxReaderConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.uniprot.store.spark.indexer.taxonomy.reader;
+
+public class TaxReaderConstants {
+
+    public static final String READ = "read";
```

Comment:
Can you please let me know if you still need this constant? You can use TaxDb now, right?